lib/chibi/test.scm

```diff
@@ -23,6 +23,20 @@
             args)
   (newline (current-error-port)))
 
+(define (exception-message exc)
+  (let* ((s (parameterize
+               ((current-output-port (open-output-string)))
+             (print-exception exc (current-output-port))
+             (get-output-string (current-output-port))))
```

Comment:
There's no need to parameterize the current-output-port, you can just create a local string port.

Reply:
Implemented